src/Symfony/Bundle/WebProfilerBundle/Tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -60,8 +60,6 @@ public function getDebugModes()
     }
 
     /**
-     * @group legacy
```

Comment:
Removing this was missed in #33507 where the deprecation of the `intercept_redirects` option was reverted.